### PR TITLE
Updating the home page with all cards about to be published

### PR DESCRIPTION
**Pull request by ricardozanini**
**Description:**
In this PR, I updated the home page with all the remaining cards for the PRs to be published. Also updated the PR template to remind the authors to update this file.

Must be merged after:

- #29 
- #45 
- #55 
- #54 
- #52 
- #47 
- #46 
- #37 
- #34 
- #32 
- #30 
- #29 

Please make sure that your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -90,5 +165,35 @@ Testing your Serverless Workflow application using REST Assured
...
+
+[.card-section]
+== Integrations
+
+[.card]
+--
+[.card-title]
+xref:integrations/serverless-dashboard-with-runtime-data.adoc[Displaying workflow runtime data in Serverless Dashboards]
+[.card-description]
+Learn how to use dashboards to display the runtime data of your workflow applications
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:198]: maybe add the "Found a issue" here?

**Review by spolti — APPROVED** [review]

**Review by davidesalerno — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -79,7 +138,23 @@ Authentication for OpenAPI services in Serverless Workflow
...
+Learn how to do integration tests on workflow applications that use PostgreSQL as persistence storage
+--
+
+[.card]
+--
+[.card-title]
+xref:testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc[Mocking OpenAPI services with WireMock]
+[.card-description]
+Learn how to mock external REST requests when testing your workflow applications
 --
```
Comment (nmirasch) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:159]: @ricardozanini  I miss this guide linking: xref:testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc[Mocking HTTP CloudEvents Sink with Wiremock] even the related PR is mentioned as included by this one. Should I add it in the guide PR or will be added by this one?

Comment (ricardozanini) [reply serverlessworkflow/modules/ROOT/pages/index.adoc:159]: I thought that you already included it in your PR, that's why I haven't added it here.

Comment (nmirasch) [reply serverlessworkflow/modules/ROOT/pages/index.adoc:159]: @ricardozanini I just want to sync my PR with this one, Thanks!

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -8,15 +8,15 @@
 [.card-title]
 xref:getting-started/create-your-first-workflow-service.adoc[Create your first workflow service]
 [.card-description]
-Create your first workflow service
+Learn how to create your firt workflow project from the ground up
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:11]:
```suggestion
Learn how to create your first Serverless Workflow project 
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -8,15 +8,15 @@
...
 --
 
 [.card]
 --
 [.card-title]
 xref:getting-started/cncf-serverless-workflow-specification-support.adoc[CNCF Serverless Workflow specification]
 [.card-description]
-CNCF Serverless Workflow specification
+Learn the details about the CNCF Serverless Workflow Specification implementation
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:19]:
```suggestion
Learn about the CNCF Serverless Workflow Specification implementation
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -27,7 +27,23 @@ CNCF Serverless Workflow specification
 [.card-title]
 xref:core/custom-functions-support.adoc[Custom functions for your Serverless Workflow service]
 [.card-description]
-Custom functions for your Serverless Workflow service
+Learn more about the custom functions supported by Kogito Serverless Workflow beyond the specification
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:30]:
```suggestion
Learn about the custom functions supported by Serverless Workflow
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -27,7 +27,23 @@ CNCF Serverless Workflow specification
...
+--
+
+[.card]
+--
+[.card-title]
+xref:core/understanding-jq-expressions.adoc[Understanding JQ expressions]
+[.card-description]
+Learn how to create jq expressions to manipulate data within a workflow execution
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:38]:
```suggestion
Learn how to create jq expressions to manipulate data within a Serverless Workflow execution
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -27,7 +27,23 @@ CNCF Serverless Workflow specification
...
+--
+
+[.card]
+--
+[.card-title]
+xref:core/understanding-workflow-error-handling.adoc[Error handling in Serverless Workflow]
+[.card-description]
+Learn how to handle errors in your workflow application
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:46]:
```suggestion
Learn how to handle errors in your Serverless Workflow application 
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -38,7 +54,23 @@ Custom functions for your Serverless Workflow service
...
-Serverless Workflow editor
+Learn how to install and use the Serverless Workflow editor
+--
+
+[.card]
+--
+[.card-title]
+xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc[Kogito Serverless Workflow Tools extension in Quarkus Dev UI]
+[.card-description]
+Learn how to use the Kogito Serverless Workflow extension in Quarkus Dev UI
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:65]:
```suggestion
Learn how to use the Serverless Workflow extension in Quarkus Dev UI
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -38,7 +54,23 @@ Custom functions for your Serverless Workflow service
...
+--
+
+[.card]
+--
+[.card-title]
+xref:tooling/kn-plugin-workflow-overview.adoc[Workflow plugin for Knative CLI]
+[.card-description]
+Learn how to install the workflow plugin for Knative CLI
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:73]:
```suggestion
Learn how to install the Serverless Workflow plugin for Knative CLI
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 10]
```diff
@@ -49,15 +81,34 @@ Serverless Workflow editor
 [.card-title]
 xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
 [.card-description]
-Orchestrating the OpenAPI services
+Learn more about orchestrating REST services using OpenAPI specification descriptors
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:84]:
```suggestion
Learn how to orchestrate REST services using OpenAPI specification descriptors
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 11]
```diff
@@ -49,15 +81,34 @@ Serverless Workflow editor
...
 --
 
 [.card]
 --
 [.card-title]
 xref:service-orchestration/orchestration-of-grpc-services.adoc[Orchestration of gRPC based services]
 [.card-description]
-Orchestration of gRPC based services
+Learn more about orchestrating gRPC services
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:92]:
```suggestion
Learn about orchestrating gRPC services
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 12]
```diff
@@ -49,15 +81,34 @@ Serverless Workflow editor
...
+
+[.card-section]
+== Eventing
+
+[.card]
+--
+[.card-title]
+xref:eventing/working-with-callbacks.adoc[Working with Callbacks]
+[.card-description]
+Learn how to use the Callback state in your workflow application
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:103]:
```suggestion
Learn how to use the Callback state in your Serverless Workflow application
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 13]
```diff
@@ -49,15 +81,34 @@ Serverless Workflow editor
...
+--
+
+[.card]
+--
+[.card-title]
+xref:eventing/consume-producing-events-with-kafka.adoc[Consuming and producing events with Kafka]
+[.card-description]
+Learn how to configure your workflow application to produce and consume events with Kafka
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:111]:
```suggestion
Learn how to configure your Serverless Workflow application to produce and consume events using Kafka
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 14]
```diff
@@ -68,7 +119,15 @@ Orchestration of gRPC based services
 [.card-title]
 xref:security/authention-support-for-openapi-services.adoc[Authentication for OpenAPI services in Serverless Workflow]
 [.card-description]
-Authentication for OpenAPI services in Serverless Workflow
+Learn how to use authentication methods when calling REST services via OpenAPI specification
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:122]:
```suggestion
Learn how to use authentication methods when calling REST services using OpenAPI specification
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 15]
```diff
@@ -68,7 +119,15 @@ Orchestration of gRPC based services
...
+--
+
+[.card]
+--
+[.card-title]
+xref:security/orchestrating-third-party-services-with-oauth2.adoc[Orchestrating third party services with OAuth2 authentication]
+[.card-description]
+Learn more about the OAuth2 method support when orchestrating REST services with your workflow applications
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:130]:
```suggestion
Learn about the OAuth2 method support when orchestrating REST services using your Serverless Workflow application
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 16]
```diff
@@ -79,7 +138,23 @@ Authentication for OpenAPI services in Serverless Workflow
 [.card-title]
 xref:testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc[Testing your Serverless Workflow application using REST Assured]
 [.card-description]
-Testing your Serverless Workflow application using REST Assured
+Learn how to add unit tests in your workflow application using RestAssured
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:141]:
```suggestion
Learn how to add unit tests in your Serverless Workflow application using RestAssured
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 17]
```diff
@@ -79,7 +138,23 @@ Authentication for OpenAPI services in Serverless Workflow
...
+--
+
+[.card]
+--
+[.card-title]
+xref:testing-and-troubleshooting/integration-tests-with-postgresql.adoc[Integration Test with PostgreSQL]
+[.card-description]
+Learn how to do integration tests on workflow applications that use PostgreSQL as persistence storage
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:149]:
```suggestion
Learn how to integrate tests on Serverless Workflow applications that use PostgreSQL as a persistence storage
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 18]
```diff
@@ -79,7 +138,23 @@ Authentication for OpenAPI services in Serverless Workflow
...
+--
+
+[.card]
+--
+[.card-title]
+xref:testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc[Mocking OpenAPI services with WireMock]
+[.card-description]
+Learn how to mock external REST requests when testing your workflow applications
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:157]:
```suggestion
Learn how to mock external REST requests when testing your Serverless Workflow applications
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 19]
```diff
@@ -90,5 +165,35 @@ Testing your Serverless Workflow application using REST Assured
 [.card-title]
 xref:cloud/build-workflow-image-with-quarkus-cli.adoc[Building Serverless Workflow Images using Quarkus CLI]
 [.card-description]
-Building Serverless Workflow Images using Quarkus CLI
+Learn how to build images for your workflow applications using Quarkus CLI
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:168]:
```suggestion
Learn how to build images for your Serverless Workflow applications using Quarkus CLI
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 20]
```diff
@@ -90,5 +165,35 @@ Testing your Serverless Workflow application using REST Assured
...
+--
+
+[.card]
+--
+[.card-title]
+xref:cloud/deploying-on-minikube.adoc[Deploying your Serverless Workflow application on Minikube]
+[.card-description]
+Learn how to deploy your workflow application on Minikube for local tests and development
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:176]:
```suggestion
Learn how to deploy your Serverless Workflow application on Minikube for local tests and development
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 21]
```diff
@@ -90,5 +165,35 @@ Testing your Serverless Workflow application using REST Assured
...
+
+[.card-section]
+== Use Cases
+
+[.card]
+--
+[.card-title]
+xref:use-cases/orchestration-based-saga-pattern.adoc[Orchestration-based SAGA pattern]
+[.card-description]
+Learn how and when to use the SAGA pattern in your workflow projects
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:187]:
```suggestion
Learn how and when to use the SAGA pattern in your Serverless Workflow projects
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:198]:
```suggestion
Learn how to use dashboards to display the runtime data of your Serverless Workflow application
```

**Review by hmanwani-rh — COMMENTED** [review]
@ricardozanini Added some suggestions to the brief descriptions. Please take a look and let me know if you've any questions.

**Comment by ricardozanini** [comment]
Thanks, @hmanwani-rh. Your changes have been incorporated. I'll wait for all the PRs to be merged to merge this one.

**Review by krisv — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 22]
```diff
@@ -68,7 +119,15 @@ Orchestration of gRPC based services
...
 xref:security/authention-support-for-openapi-services.adoc[Authentication for OpenAPI services in Serverless Workflow]
 [.card-description]
-Authentication for OpenAPI services in Serverless Workflow
+Learn how to use authentication methods when calling REST services using OpenAPI specification
+--
+
+[.card]
+--
+[.card-title]
+xref:security/orchestrating-third-party-services-with-oauth2.adoc[Orchestrating third party services with OAuth2 authentication]
```
Comment (wmedvede) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:128]:
```suggestion
xref:security/orchestrating-third-party-services-with-oauth2.adoc[Orchestration of third-party services using OAuth 2.0 authentication]
```

**Review by wmedvede — COMMENTED** [review]
I've just added @hmanwani-rh suggestion to the oauth2 security card.

**Comment by ricardozanini** [comment]
We only have two left that are likely to be merged soon and we need to alert the contributions for the need to add the cards in each PR. Merging.

**Pull request merged**